jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -86,7 +89,6 @@ spec:
             - name: https
               containerPort: 443
               protocol: TCP
-        {{ if and .Values.proxy.https.hosts (and .Values.proxy.https.enabled (eq .Values.proxy.https.type "letsencrypt" ) ) -}}
```

Comment:
missing `if $autoHTTPS` here?

Reply:
I just realized this is a rat's nest of conditionals. I'm just going to split the autoHTTPS part into its own deployment, which should make things easier for us to maintain for now. Longer term, we can replace CHP with something that does ACME (or add a sidecar that does ACME and stores the certificates in a kubernetes Secret object)